docs/environments.md

```diff
@@ -1,6 +1,60 @@
 # Deployment Environments
 
-This section describes the different environments used to deploy all the Thoth Services to. All environments should be deployed using ArgoCD.
+This section describes the different environments used to deploy all the Thoth Services to OpenShift Cluster.
+
+*Note*: All environments can be deployed using ArgoCD or Kustomize.
+
+
+## Database Setup
+
+Project Thoth uses Postgresql database for knowlegde graph storage.
+
+- Deployment manifest can be found at: [Graph Manifest](../core/overlays/cnv-prod/graph-prod)
+- we manage the storage via different project Thoth components which utlises the [thoth-storage](https://pypi.org/project/thoth-storages/) module. [Details](https://github.com/thoth-station/storages#thoth-storages)
+- Local setup and management information can be acquired at [thoth-storages](https://github.com/thoth-station/storages#thoth-storages)
+
+Extensions:
+
+**amcheck**: we have setup amcheck extension for detecting database corruption errors.
+The [Issue](https://github.com/thoth-station/storages/issues/1586) based on which it was decided to use this extension.
+
+The check for database corruption is executed from metrics-exporter.
+The setup of the extension is to be performed with setup of database. init-job would help if being used, if manual process is to be used then perform the following action:
+
+Execute `CREATE EXTENSION amcheck;` in the database where amcheck is to be set. We rsh into the pod handling the postgresl and perform following action:
+
+```bash
+$ psql
+psql (10.15)
+Type "help" for help.
+
+postgres=# \c thoth
+You are now connected to database "thoth" as user "postgres".
+thoth=# CREATE EXTENSION amcheck;
+CREATE EXTENSION
+```
+
+It can be verified if installed via `\dx`
+
+```bash
+thoth=# \dx
+                                    List of installed extensions
+        Name        | Version |   Schema   |                        Description
+--------------------+---------+------------+-----------------------------------------------------------
+amcheck            | 1.0     | public     | functions for verifying relation integrity
```

Comment:
is this somethign we have declared in some yaml? or can put to a file in a repo?

Reply:
we can set this from a component that talks to storage.
so we will have this in init-job, so some newly setup their environment can benefit from it.